src/qiskit_iqm/fake_backends/chip_samples/example_sample.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 Qiskit on IQM developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+    This module shows how to create a chip sample. Copy this file and populate it with your measured values.
+"""
+from .. import IQMChipSample
+from ..quantum_architectures import Adonis
+
+adonis_chip_sample = IQMChipSample(
```

Comment:
This should either contain realistic values (if not confidential) or should not exist at all, since otherwise it is confusing. This is basically just a dummy example of how to create an `IQMChipSample` instance, which should be clear from documentation already

Reply:
If it is realistic parameters, then I still don't think it warrants its own sub-package - it could just be part of `FakeAdonis` class.

Reply:
un-resolving since not completely sure how this was resolved. Comment pls.

Reply:
The file does not exist and it is in the same file that contaisn the FakeAdonis definition. The parameters are realistic ones now